src/WidgetIncluderModule.php

```diff
@@ -21,53 +21,64 @@ public function main(App $app)
 
     public function renderWidget(App $app, $widget_id, $options = [], $view = null)
     {
+
```

Comment:
I would remove this empty line. It is not required and not always there. So for consistency remove it (also pagekit has it not there in the places I checked).